### PR TITLE
Setting for getting mentor request

### PR DESCRIPTION
**Pull request by SarahhhC**
* User setting to decide getting mentor request.
* Unit test added.

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -380,10 +379,31 @@ export function editPersonality(req, res, next) {
...
         res.status(400).json(err);
       });
   } else {
     res.status(401).json({ err_point: userCallback.ERR_FAIL_AUTH });
   }
 }
+
+export function mentorRequestSetting(req, res, next) {
+  if (req.session._id) {
+    if (req.params.flag === 'true' || req.params.flag === 'false') {
```
Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: Does the flag parameter can have another options?

Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: No, only true or false would be valid.

File: src/server/models/users.model.js [hunk 2]
```diff
@@ -81,6 +81,7 @@ let userSchema = new Schema({
   job_position: String,
   about: String,
   languages: String,
+  requestGet: Boolean,
```
Comment (KyuaKwon) [inline src/server/models/users.model.js]: What is this value for?

Comment (KyuaKwon) [inline src/server/models/users.model.js]: One of my mentor or not?

Comment (SarahhhC) [inline src/server/models/users.model.js]: This is for decision whether user get the mentor request or not.
If value is false, user will not receive any mentor request.

Comment (KyuaKwon) [inline src/server/models/users.model.js]: I think the name of this value need to be more accurate.
Now is little bit ambiguous to me.

Comment (SarahhhC) [inline src/server/models/users.model.js]: I like first one! Thanks for the recommendation!!!

Comment (KyuaKwon) [inline src/server/models/users.model.js]: Wow~~

**Comment by KyuaKwon** [comment]
:shipit: 😉

**Pull request merged**